Remotion/Web/Core/Res/Themes/NovaViso/HTML/SingleView.css

```
@@ -1,79 +1,85 @@
 /* Style applied to the SingleView. */
 
+:root
+{
+  --singleview-topcontrols-padding: 0;
```

```suggestion
  --singleview-topcontrols-padding: var(--remotion-themed-spacing) var(--remotion-themed-spacing-large);
```
for the default, for now.